Geometry_Engine/Query/LineIntersections.cs

```diff
@@ -263,6 +304,12 @@ public static List<Point> LineIntersections(this PolyCurve curve, Line line, boo
 
         /***************************************************/
 
+        [Description("Calculates and returns the intersection points of two Polylines.")]
+        [Input("curve1", "First Polyline to intersect with the second Polyline.")]
+        [Input("curve2", "Second Polyline to intersect with the first Polyline.")]
+        [Input("tolerance", "Distance tolerance to be used in the method. Used for checking if the intersection point is within acceptable distance from the Lines.", typeof(Length))]
+        [Input("angleTolerance", "Angle tolerance to be used in the method. Used for checking if the lines are colinear", typeof(Angle))]
```

Comment:
First of all, why is this method named `LineIntersections` if it is checking intersections between `Polylines`...?
```suggestion
        [Input("angleTolerance", "Angle tolerance to be used in the method. Used for checking if the polyline segments are colinear.", typeof(Angle))]
```

Reply:
Assume it is because it is doing linear intersections, but get what you mean.

WOuld prefer to not do any renaming in this PR though